rllib/models/torch/misc.py

```diff
@@ -15,7 +15,7 @@ def initializer(tensor):
     return initializer
 
 
-def valid_padding(in_size, filter_size, stride_size):
+def same_padding(in_size, filter_size, stride_size):
```

Comment:
The old name was VERY confusing :) being "valid_padding", where we wanted to imitate tf's "same" behavior.